README.md

```diff
@@ -56,7 +56,7 @@ Alternatively, you can add the Microsoft.ML package from within Visual Studio's
 
 Daily NuGet builds of the project are also available in our Azure DevOps feed:
 
-> [https://pkgs.dev.azure.com/dnceng/public/_packaging/MachineLearning/nuget/v3/index.json](https://pkgs.dev.azure.com/dnceng/public/_packaging/MachineLearning/nuget/v3/index.json)
+> [https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries/nuget/v3/index.json](https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries/nuget/v3/index.json)
```

Comment:
This changing the internal feed location. right? I am just asking as sometimes I share that with other parties.

Reply:
Yes, we discussed this with the arcade folks and they said we shouldn't maintain our own Artifacts feed.  They wanted us to use the common dotnet-libraries feed just like other libraries similar to us.  For example: https://github.com/dotnet/command-line-api/blob/main/README.md

The actual change that moved the feed was in https://github.com/dotnet/machinelearning/commit/de79b8a1b765f77ae46a91e12c83b8e4543fbf1e#diff-70773eb1e66297b0e83e4f7d57285bec90b978c662da1c90a5fcbcb1bc534688L346

If we wanted to bring back publishing to the other feed we can, but my goal is to get us as much as possible off "special cases"